pallets/duster/src/lib.rs

```diff
@@ -323,3 +323,24 @@ impl<T: Config> Contains<T::AccountId> for DusterWhitelist<T> {
 		AccountBlacklist::<T>::contains_key(t)
 	}
 }
+
+use hydradx_traits::pools::DustRemovalAccountWhitelist;
+
+impl<T: Config> DustRemovalAccountWhitelist<T::AccountId> for Pallet<T> {
```

Comment:
it is blacklist or whitelist? and shouldn't this be integrated other way around with handler in duster instead of XYK?

Reply:
i am aware of the inconsistency. 

it should really be whitelist. orml-tokens uses whitelist, so i would like to be consistent with that .

however, renaming the storage from blacklist to whitelist would require migration - which i dont really want to do right now. 

It can be as part of another task/issue.

Reply:
and regarding the other suggestion.

i was thinking how to do - and one way i was considering was the other way.

but that's not really duster feature to include "other stuff" and it is reallysomething that XYK want to force its user/consumer to do . 

by doiing this - forcing to configure XYK pallet in a way that pool accounts will be excluded ( or should be excluded ) from dusting , you achieve what XYK pallet needs to have for pool accounts. 

otherwise - it XYK pallet would be used elsewhere - there is no way to make sure what needs to be done with this.